examples/authorization_details_ldp_vc.json

```diff
@@ -3,11 +3,11 @@
         "type": "openid_credential",
         "credential_configuration_id": "UniversityDegree_LDP_VC",
         "credential_definition": {
-            "credentialSubject": {
-                "given_name": {},
-                "family_name": {},
-                "degree": {}
-            }
+            "credentialSubject": [
```

Comment:
`credentialSubject` -> `credential_subject`

If the current evolution of this project introduces breaking changes, this presents an opportune moment to standardize naming conventions across the board. This could mean adopting snake_case.

Reply:
True. I had the same thought. Although I'm inclined to go for `claims` instead.

Reply:
Microsoft just finished implementing with 'credentialSubject'. Given this is more of an issue of preference, would appreciate if we could keep this camel case.

Reply:
It appears at times that, on behalf of your employer, you may be hindering the community from considering better specifications. This is a significant concern for the community.

Reply:
They would need to change quite a lot around the processing anyways, I can't imagine how a single renamed field would add significant cost to that.

Reply:
Having disparate casings in a standard is frankly infuriating. Don't do that :)

Reply:
@peppelinux @TakahikoKawasaki @vdzhuvinov This has been resolved with my latest change to this PR: https://github.com/openid/OpenID4VCI/pull/276#discussion_r1860262845